<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmtpSimpleSettingsWidget</class>
 <widget class="QWidget" name="SmtpSimpleSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>135</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="emailLabel">
     <property name="text">
      <string>&amp;Email:</string>
     </property>
     <property name="buddy">
      <cstring>emailLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="emailLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>&amp;Password:</string>
     </property>
     <property name="buddy">
      <cstring>passwordLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="signatureLabel">
     <property name="text">
      <string>&amp;System Signature:</string>
     </property>
     <property name="buddy">
      <cstring>signatureLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="signatureLineEdit">
     <property name="placeholderText">
      <string>Enter a short system description here.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="supportEmailLabel">
     <property name="text">
      <string>Support Signature:</string>
     </property>
     <property name="buddy">
      <cstring>supportLinkLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="supportLinkLineEdit"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="detectErrorLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
